marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/step/getStepsByType.sjs

```diff
@@ -24,7 +24,7 @@ const Artifacts = require('/data-hub/5/artifacts/core.sjs');
 var propertiesToReturn;
 
 const response = {};
-const stepTypes = ['ingestion', 'mapping'];
+const stepTypes = ['ingestion', 'mapping', 'matching', 'merging', 'mastering', 'custom'];
```

Comment:
Given that we want all steps, it'll be simpler and faster to just query for all steps (they all share a collection) and then group them into a sorted map structure. That's 1 query instead of 6 queries, and also ensures we don't need to update this again (stepTypes I believe only existed before as a way of preventing anything besides ingestion/mapping from showing up).

Reply:
Done